osc.c

```diff
@@ -1463,7 +1464,11 @@ GtkWidget * new_plot_cb(GtkMenuItem *item, gpointer user_data)
 {
 	GtkWidget *new_plot;
 
-	new_plot = osc_plot_new(ctx);
+	if (osc_preferences && osc_preferences->plot_preferences)
```

Comment:
the use of the global `osc_preferences` variables looks a bit weird here;
a instance is dependent on the existence of the global var;

at least in the current context, i don't see why this is needed [of it is really needed]

Reply:
my fault here;
i misinterpreted the `osc_preferences` var-name